src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -324,7 +324,7 @@ To find the key for existing private location:
 
 ## Install, update, and configure synthetics job manager [#install]
 
-As you start setting up job managers, make sure you do the following to avoid port contention:
+If you're running more than one Docker private location container in the same host, you'll have port conflicts. To avoid this port contention, you'd make sure you do the following when you start setting up job managers:
```

Comment:
@nbaenam I'd suggest you drop the "would" verb form and go with this:

If you're running more than one Docker private location container in the same host, you'll have port conflicts. To avoid this port contention, make sure you do the following when you start setting up job managers:

Reply:
Good catch! Thank you!